style.css

```diff
@@ -101,13 +101,245 @@ body {
   top: 120px;
 }
 
-.portfolio p {
+.portfolio-heading {
   font-family: 'Crete Round', serif;
   font-weight: 400;
   font-size: 40px;
+  text-align: left;
   line-height: 52px;
-  margin: 120px 24px;
+  margin-top: 120px;
+}
+
+.portfolio-img {
+  margin-top: 72px;
+}
+
+#portfolio {
+  color: #091e42;
+  margin-left: 24px;
+  margin-right: 24px;
+  margin-bottom: 24px;
+}
+
+.portfolio-text {
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 1rem;
+  line-height: 24px;
+}
+
+.rule {
+  margin-top: 24px;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-rows: 880px 386px 386px;
+  grid-template-columns: 375px;
+  gap: 48px;
+}
+
+.skill-list {
+  list-style: none;
+  display: flex;
+  margin-right: 12px;
+  padding-left: 0;
+  margin-bottom: 24px;
+}
+
+.skill-list li {
+  border: solid 1px;
+  padding: 10px 12px;
+  margin-right: 12px;
+  line-height: 20px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 500;
+  font-size: 15px;
+}
+
+.grid-container a button {
+  text-decoration: none;
+  color: #fff;
+  padding: 12px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  line-height: 24px;
+  font-size: 15px;
+  height: 48px;
+  cursor: pointer;
+}
+
+.skill-list-btn {
+  text-decoration: none;
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  line-height: 24px;
+  font-size: 16px;
+  width: 124px;
+  height: 48px;
+  cursor: pointer;
+  padding: 12px;
+}
+
+.btn {
+  background-color: #ff6b00;
+  border: none;
+}
+
+.printing {
+  background-image: url('images/background.png');
+  height: 386px;
+  margin: 0 24px;
+}
+
+.printing h2 {
+  margin-top: 84px;
+  font-family: 'Crete Round', serif;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  margin-left: 16px;
+  margin-right: 16px;
+  color: #fff;
+}
+
+#portfolio h2 {
+  font-family: 'Crete Round', serif;
+  font-size: 32px;
+  line-height: 44px;
+  font-weight: 400;
+}
+
+.btn_2 {
+  width: 327px;
+  padding: 12px 118.7px;
+  height: 48px;
+  text-decoration: none;
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  line-height: 24px;
+  font-size: 16px;
+}
```

Comment:
- [ ] Kindly ensure that your button color effects for hover, active (pressed) and disabled are all implemented as we have in the Figma design

  ![buttons-color](https://user-images.githubusercontent.com/36542986/159539913-8d731507-699b-4238-b619-d36566162328.jpg)